### DISCUSSION TITLE
No alerts from Elastic agent

### DISCUSSION THREAD

**Discussion by Chizan-Dolrocks** [Security-Onion-Solutions/securityonion#11937] [Unsupported Versions] [answered]
### Hello,
I just wanted to know if I'm the only one having this problem: When I go into the Elastic Fleet manager and click on an agent to click on "View more agent metrics" The links on the left in the "Agent Health" section such as "Overview", "Agent Information", "Agent Metrics" or "Integrations" lead to a page that contains _no data._

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84504744/c03b1bce-7432-4002-ae26-013efd47286d)

**Thank you**.

**Comment by weslambert** [comment]
What do you mean by alerts? We recommend reviewing alerts inside of SOC Alerts.

The Elastic Agent dashboards haven't been tested extensively. 

I don't think we collect metrics by default because of the potential immense volume and requirement for storage, so there may not be data to populate the Metrics dashboard.

I have data on the Integrations page.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/16829864/86f11059-b7a4-4799-85ff-d2bbd344005e)

**Comment by Chizan-Dolrocks** [comment]
It's just that we don't have alerts in SOC,  but we have data on the Integration page too.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84504744/e166ede8-3587-4c68-a800-1c033f016b90)

**Comment by weslambert** [comment]
From the picture it looks like you have data from Suricata, so you should have alert data provided that you are not using Suricata for metadata and have Zeek data.

Do you see any data is you use the following query in SOC Hunt?

event.module:suricata

**Reply by Chizan-Dolrocks** [reply]
Sorry for the delay, but I have suricata alerts in SOC : 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84504744/63aa5c39-be6f-44c6-a700-b141c9e1998a)

but I just don't get the Elastic Agent alerts in SOC.

in kibana we have this :  <html><body>

error.message  | failed in Fleet agent final_pipeline: Text '2023-12-24T07:36:54.599Z' could not be parsed at index 20
-- | --



</body>
</html>

**Comment by dougburks** [comment, marked as answer]
Have you tried going to Playbook and activating plays for alerts that you would want to see?
https://docs.securityonion.net/en/2.4/playbook.html

**Reply by Chizan-Dolrocks** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/84504744/60164970-3fab-4489-98d8-1d5a64f4cdfd)

Now I have alert from playbook, but I wanted to know if I could get alerts from elastic agent like event.module

Thank you

**Reply by dougburks** [reply]
Playbook is running queries against your Elastic Agent data.
